repository/spark/operator/params.yaml

```diff
@@ -114,6 +114,11 @@ parameters:
     default: ""
     displayName: "Spark History Server Persistent Volume Claim Name"
 
+  - name: historyServerSparkConfSecret
+    description: "Name of a Secret with Spark config file. The secret should be in the same namespace as the Operator."
+    default: ""
+    displayName: "Name of a Secret with AWS credentials"
```

Comment:
```suggestion
    displayName: "Name of a Secret with Spark History Server configuration file"
```

Reply:
good catch, thanks! applying the suggestion